lax/lichens/sciencerun0.py

```diff
@@ -262,9 +262,10 @@ class FiducialZOptimized(StringLichen):
 
     https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:analysis:sciencerun1:summary_fiducial_volume
     """
-    version = 0
-    string = "(-95 < z_3d_nn) & (z_3d_nn < -7) & \
-              (z_3d_nn < 27.2929 - 0.0276385*r_3d_nn*r_3d_nn)"
+    version = 1
+    string = "(-95 < z_3d_nn) & (z_3d_nn < -8) & \
```

Comment:
Where's the R cut?

Reply:
It's logical 'and' between all 4 lines.

Reply:
Added in https://github.com/XENON1T/lax/pull/145/commits/9e9eeebcfd8698b13768bda543b555127c600699: ```r_3d_nn < 42.8387```